docs/en-US/component/pagination.md

```diff
@@ -78,6 +78,7 @@ pagination/more-elements
 | default-current-page                | default initial value of current-page, not setting is the same as setting 1                                                     | ^[number]                                                                         | —                                    |
 | layout                              | layout of Pagination, elements separated with a comma                                                                           | ^[string]`string (consists of sizes, prev, pager, next, jumper, ->, total, slot)` | prev, pager, next, jumper, ->, total |
 | page-sizes                          | options of item count per page                                                                                                  | ^[object]`number[]`                                                               | [10, 20, 30, 40, 50, 100]            |
+| append-size-dropdown-to             | which element the size dropdown appends to                                                                                      | ^[string]                                                                         | body                                 |
```

Comment:
The prop name should be shorter, such as `size-append-to`.

Reply:
cc @tolking @ryuhangyeong @kooriookami

Reply:
Okay, I'll change it